pyextract.py

```diff
@@ -233,7 +233,8 @@ def __find_logfiles_path__(self):
     def __find_coredump(self):
         for root, dirs, files in os.walk(self.log_dir_path):
             for file in files:
-                if file.startswith(DefaultCLIParameters.core_prefix):
+                # find core dump file by prefix "core-" or "Core-"
```

Comment:
This change converts both the filename and the prefix to lowercase before checking if the filename starts with the prefix. This makes the check case-insensitive, allowing it to match files regardless of whether the prefix is in uppercase, lowercase, or a mix of both.

Reason for Change:
- Ensures that core dump files with prefixes like `core-` or `Core-` are found, regardless of the case.
- Improves the robustness of the file search by handling different file naming conventions.